apps/tailwind-components/components/Button.vue

```diff
@@ -55,22 +58,27 @@ const colorClasses = computed(() => {
 });
 
 const sizeClasses = computed(() => {
-  return SIZE_MAPPING[props.size];
+  return props.iconOnly ? "p-[8px]" : SIZE_MAPPING[props.size];
 });
 
 const iconPositionClass = computed(() => {
   return ICON_POSITION_MAPPING[props.iconPosition];
 });
+
+const tooltipText = computed(() => {
+  return props.tooltip || props.iconOnly ? props.label : "";
+});
```

Comment:
Is there a way to make the label required if the `iconOnly` prop is true? Alternatively, there could be a check here that throws a warning that the label is missing or blank. Otherwise, the context is lost if the default value for label is `""`.